src/sdk/base/src/main/java/owt/base/PeerConnectionChannel.java

```diff
@@ -219,11 +213,11 @@ protected void removeStream(String mediaStreamId) {
                 return;
             }
             Log.d(LOG_TAG, "remove stream");
-            if (audioRtpSenders.get(mediaStreamId) != null) {
-                peerConnection.removeTrack(audioRtpSenders.get(mediaStreamId));
-            }
-            if (videoRtpSenders.get(mediaStreamId) != null) {
-                peerConnection.removeTrack(videoRtpSenders.get(mediaStreamId));
+            for(RtpSender sender: peerConnection.getSenders()) {
+                List<String> streams = sender.getStreams();
+                if (streams.size() == 1 && streams.get(0) == mediaStreamId) {
```

Comment:
stream.size() >=1?

Reply:
`streams.size()>1` means the sender is shared by multiple streams. Stopping the sender may stop the track in other MediaStreams as well.  A better way is to remove `mediaStreamId` from `sender.getStreams()` list, but I'm not sure if it causes new issues. Moving to track based APIs would eventually fix this issue.